packages/launcher/src/index.tsx

```diff
@@ -379,14 +366,16 @@ function Card(kernel: boolean, item: ILauncherItem, launcher: Launcher, launcher
       return;
     }
     launcher.pending = true;
-    let callback = item.callback as any;
-    let value = callback(launcher.cwd, item.name);
+    let value = commands.execute(command, {
```

Comment:
`value` is a `Promise`, so it should probably be called something else, maybe `promise`, and the `Promise.resolve(value).then(...)` line that follows should probably be `promise.then(...)`